app/src/main/java/org/simple/clinic/settings/SettingsRepositoryImpl.kt

```diff
@@ -1,12 +1,31 @@
 package org.simple.clinic.settings
 
+import com.f2prateek.rx.preferences2.Preference
 import io.reactivex.Completable
+import io.reactivex.Maybe
 import io.reactivex.Single
+import org.simple.clinic.util.Optional
+import org.simple.clinic.util.filterAndUnwrapJust
+import java.util.Locale
 
-class SettingsRepositoryImpl : SettingsRepository {
+class SettingsRepositoryImpl(
```

Comment:
Can we use a different name instead? `PreferencesSettingsRepository`?

Reply:
Done.